lib/matplotlib/backend_bases.py

```diff
@@ -1528,7 +1528,6 @@ class Done(Exception):
     def _draw(renderer): raise Done(renderer)
 
     with cbook._setattr_cm(figure, draw=_draw), ExitStack() as stack:
-        orig_canvas = figure.canvas
```

Comment:
This is unused and I happened to stumble upon it.

Reply:
Usage of `orig_canvas` was removed in c712f35eb, so that line can go. I suggest to put this in a separate PR. This PR is quite large but mostly mechanical replacement. If you start to mix other aspects in, that makes review more cumbersome. - Yes also a deletion of a single unused variable can be a dedicated PR. You may mix it together with other small cleanups if you have some anyway.

Reply:
Yeah, I realized the potential problem which is why I mentioned it. I'll try to remove it so that this hopefully can be merged soon (I have some other cleanups that depends on this PR).